assets/src/components/map.tsx

```diff
@@ -342,16 +394,36 @@ const EventAdder = ({
     // `dragstart` is fired when a user drags the map
     // it is expected that this event is not fired for anything but user input
     // by [handler/Map.Drag.js](https://github.com/Leaflet/Leaflet/blob/6b90c169d6cd11437bfbcc8ba261255e009afee3/src/map/handler/Map.Drag.js#L113-L115)
-    dragstart() {
+    dragstart: () => {
       setShouldAutoCenter(false)
     },
     // `moveend` is called when the leaflet map has finished animating a pan
-    moveend() {
+    moveend: () => {
       // Wait until the auto centering animation is finished to resume listening for user interaction.
       if (isAutoCentering.current) {
         isAutoCentering.current = false
       }
     },
+
+    ...(streetViewMode
+      ? {
+          click: (e) => {
+            window.open(
+              streetViewUrl({
+                latitude: e.latlng.lat,
+                longitude: e.latlng.lng,
+              }),
+              "_blank"
+            )
+          },
+
+          keydown: (e) => {
+            if (e.originalEvent.key === "Escape") {
+              setStreetViewMode(false)
+            }
+          },
+        }
+      : {}),
```

Comment:
questions(non-blocking):
- is there any particular reasoning for using `key: arrow function`?
- should `zoomend` be changed as well?

I'm personally of the preference of avoiding arrow functions if it's served by some other syntax, but I don't really know why lol! so I'm not strongly opposed to the change.

Reply:
Re: your first question, I'm just way more used to the arrow function syntax and honestly kind of forgot that you could also write a function as a value associated with an object key in the way that `zoomend` does it. I agree that one way or another we should make it consistent though. @KaylaBrady preferences one way or another?

Reply:
I slightly prefer this arrow function way - similar to Eddie, that is now how I'm most accustomed to reading/writing functions. If that is the predominant existing syntax of Skate then I'm for it!